src/components/HorizontalProgressBar/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './scss/HorizontalProgressBar.scss';
+
+const HorizontalProgressBar = ({ totalProgress, currentProgress }) => {
+  const widthStyle = {
```

Comment:
this would suck if it jumps from `0-50`, we don't have an animation or transition on this thing so we should look at including `transition` or something of the sort

Reply:
well.. i think my new animation cover this